forge-gui/res/cardsfolder/upcoming/razzle-dazzler

```diff
@@ -0,0 +1,10 @@
+Name:Razzle-Dazzler
```

Comment:
filename incomplete

Reply:
`-` needs to be `_` also
so `razzle_dazzler.txt`